### DISCUSSION TITLE
v3.0.0 - Device > Add Components is sometimes broken

### DISCUSSION THREAD

**Discussion by mattibefe** [netbox-community/netbox#7112] [Q&A]
Hello everybody,

we upgraded during the weekend to the new version 3.0.0.
I detected yesterday a bug that, when trying to add a component to an existing device, the field for device (to which I am trying to add a component) is sometimes prefilled and sometimes is empty.

It is regardless of the device type and regardless of the type of component I want to add. I detected the issue in our netbox environment but I could also recreate the issue in the demo environment, eventhough it is behaving a little bit different.

The issue is somewhat dependant of the device's name. But the threshold for buggy/ not buggy is different from our own netbox and the demo environment. Our threshold is somewhere in the region of astaaaa vs astxxxxxxxx and the threshold in the demo enviroment for functional/buggy is somewhere in the region of kxxxxxxxxxxxxx/kyyyyyyyyyyyyyyy device names.

Maybe someone else can recreate the issue in their own environment. I would be glad if someone could look into this issue.

Greetings
Matti

**Comment by rodvand** [comment]
This sounds like https://github.com/netbox-community/netbox/issues/7081 so hopefully your troubles are fixed in 3.0.1.
